pandas/tests/frame/indexing/test_insert.py

```diff
@@ -100,6 +100,6 @@ def test_insert_frame(self):
         # GH#42403
         df = DataFrame({"col1": [1, 2], "col2": [3, 4]})
 
-        msg = r"Expected a 1D array, got an array with shape \(2, 2\)"
+        msg = "Expected a one-dimensional object, got a DataFrame with 2 instead."
```

Comment:
"2" here sounds like it is referring to 2 dimensions, which is accurate but i think not intended

Reply:
Yikes yes, I forgot to add columns, thx